vcontrol/vcontrol.py

```diff
@@ -153,48 +153,61 @@ def __init__(self, port=8080, host="0.0.0.0"): # pragma: no cover
             print "Error loading environment."
             sys.exit(1)
 
-        if not env in ['docker']:
+        # VCONTROL_ENV is only 'docker' if vcontrol-daemon is run in a container
+        if env != 'docker':
             # check for docker
             try:
-                print "...found docker"
                 docker = subprocess.call("which docker", shell=True)
                 if docker != 0:
                     print "You must have docker to run vcontrol. Please install docker."
                     sys.exit(1)
+                print "...found docker"
             except Exception:
                 print "Error checking for docker. Do you have docker installed?"
                 sys.exit(1)
 
             # check for docker-machine
             try:
-                print "...found docker-machine"
                 docker_machine = subprocess.call("which docker-machine", shell=True)
                 if docker_machine != 0:
                     print "You must have docker-machine to run vcontrol. Please install docker."
                     sys.exit(1)
+                print "...found docker-machine"
             except Exception:
                 print "Error checking for docker-machine. Do you have docker-machine installed?"
+
             # check that docker env is configured
             try:
-                print "...found DOCKER_HOST"
-                docker_env = subprocess.call("env | grep DOCKER_HOST", shell=True)
-                if docker_env != 0:
-                    print "No DOCKER_HOST environment variable set. Please set DOCKER_HOST."
-                    sys.exit(1)
+                is_dockerhost = subprocess.call("env | grep DOCKER_HOST", shell=True)
+                # check if call failed
+                if is_dockerhost != 0:
+                    print "No DOCKER_HOST environment variable set."
+                    try:
+                        subprocess.check_output("export DOCKER_HOST=tcp://localhost:8080", shell=True).strip('\n')
```

Comment:
I think exporting an invalid `DOCKER_HOST` like this will actually break things.  why is it necessary to export this, shouldn't it just be variable internal to this script that gets used later?